packages/invoices/src/OutboundShipment/DetailView/modals/BatchesTable.tsx

```diff
@@ -66,6 +66,7 @@ const BatchesRow: React.FC<BatchesRowProps> = ({ batch, label, onChange }) => {
       <BasicCell align="right">{label}</BasicCell>
       <BasicCell sx={{ width: '88px' }}>
         <ModalNumericInput
+          value={batch.quantity}
```

Comment:
I just updated the quantity in the batch which re-rendered and meant I didn't use `setValue` so added `value` to `ModalNumericInput`